lib/ReactViews/Custom/Chart/BottomDockChart.tsx

```diff
@@ -443,24 +454,26 @@ class XAxis extends React.PureComponent {
           textAnchor: "middle",
           fontFamily: "Arial"
         }}
-        // .nice() rounds the scale so that the aprox beginning and
-        // aprox end labels are shown
+        // .nice() rounds the scale so that the approximate beginning and end
+        // labels are shown.
         // See: https://stackoverflow.com/questions/21753126/d3-js-starting-and-ending-tick
         scale={scale.nice()}
+        // @ts-expect-error Wrong type for children in AxisProps vs React.
```

Comment:
I wasn't sure what to do about this error and I couldn't find a way to put the `ts-expect-error` in a place that mattered, so I pulled out the children and passed that as a separate property that I could `ts-expect-error`. This triggered a ESLint warning, which I have also ignored. I have no idea about the consequences of this.